managed/models.go

```diff
@@ -29,6 +29,41 @@ type Cluster struct {
 	BackupSchedules     []BackupScheduleInfo `tfsdk:"backup_schedules"`
 	ClusterEndpoints    types.Map            `tfsdk:"cluster_endpoints"`
 	ClusterCertificate  types.String         `tfsdk:"cluster_certificate"`
+	CMKSpec             *CMKSpec             `tfsdk:"cmk_spec"`
+}
+
+type CMKSpec struct {
+	ProviderType types.String `tfsdk:"provider_type"`
+	AWSCMKSpec   *AWSCMKSpec  `tfsdk:"aws_cmk_spec"`
+	GCPCMKSpec   *GCPCMKSpec  `tfsdk:"gcp_cmk_spec"`
+	IsEnabled    types.Bool   `tfsdk:"is_enabled"`
```

Comment:
Isn't this implicit if this structure `!= nil`? Can you specify an EAR, but have `is_enabled = false`?

Reply:
Yes, it is supported to create an EAR for existing cluster with `is_enabled = false`.